tests/drum/test_fit.py

```diff
@@ -0,0 +1,246 @@
+import os
```

Comment:
So you split test_fit_inference into fit and inference.
Don't you like the idea to have test cases by framework?
framework: fit -> inference.

Reply:
I don't know what you're asking here

Reply:
I mean why did you split test_fit_inference into test_fit and test_inference

Reply:
because they're testing different functionality.

Reply:
Ok. We can hold on with my idea.

Reply:
I don't know what that means